cmd/memcached_exporter/main.go

```diff
@@ -36,6 +39,12 @@ func main() {
 		address       = kingpin.Flag("memcached.address", "Memcached server address.").Default("localhost:11211").String()
 		timeout       = kingpin.Flag("memcached.timeout", "memcached connect timeout.").Default("1s").Duration()
 		pidFile       = kingpin.Flag("memcached.pid-file", "Optional path to a file containing the memcached PID for additional metrics.").Default("").String()
+		enableTls     = kingpin.Flag("tls.enable", "Enable TLS connections to memcached").Bool()
+		certfile      = kingpin.Flag("tls.certfile", "Client certificate file.").Default("").String()
+		keyfile       = kingpin.Flag("tls.keyfile", "Client private key file.").Default("").String()
+		cafile        = kingpin.Flag("tls.cafile", "Client root CA file.").Default("").String()
+		skipVerify    = kingpin.Flag("tls.skipverify", "Skip server certificate verification").Bool()
+		serverName    = kingpin.Flag("tls.servername", "Memcached TLS certificate servername").Default("").String()
```

Comment:
Since these flags are specific to memcached, let's call them `"memcached.tls...."`.